i18n/translations/en.json

```
@@ -449,7 +449,10 @@
 	"language.variables.empty": "No translations yet",
 
 	"language.variable.delete.confirm": "Do you really want to delete the variable for {key}?",
+	"language.variable.entries": "Entries",
+	"language.variable.entries.help": "This strings will be used for translations count. Each entry corresponds to a different case counter. For example: 0, 1, 2 and more",
```

I think this would be more fitting with the existing `language.variable.value`:

```suggestion
	"language.variable.values": "Values",
	"language.variable.values.help": "These values will be used according to the translation count. Each entry corresponds to a different case, e.g. for 0, for 1, for 2 and more.",
```